enterprise/internal/codeintel/stores/lsifstore/documentation.go

```diff
@@ -226,3 +227,75 @@ func (s *Store) scanFirstDocumentationResultID(rows *sql.Rows, queryErr error) (
 	}
 	return resultID, nil
 }
+
+// DocumentationDefinitions returns the set of locations defining the symbol found at the given path ID, if any.
+func (s *Store) DocumentationDefinitions(ctx context.Context, bundleID int, path, pathID string, limit, offset int) (_ []Location, _ int, err error) {
+	resultID, err := s.documentationPathIDToID(ctx, bundleID, pathID)
+	if resultID == "" || err != nil {
```

Comment:
```suggestion
	if  err != nil || resultID == "" {
```

Oh no I had a physical reaction for `err` not being checked first.